src/otp-input/otp-input.tsx

```diff
@@ -11,6 +11,7 @@ import { FormErrorMessage } from "src/form/form-label";
 
 export const OtpInput = ({
     id,
+    value,
```

Comment:
why not default to empty array and then u need not check for it's existence in the effect?